portfolio/src/main/webapp/script.js

```diff
@@ -35,3 +35,24 @@ function createListElement(text) {
   liElement.innerText = text;
   return liElement;
 }
+
+/** Gets login status */
+async function getLogin() {
+  getComments();
+  // hide comments
+  document.getElementById('comments-form').style.display = 'none';
```

Comment:
It might be good to move this into toggleComments(bool showComments) function. Makes the code more readable.